### DISCUSSION TITLE
Errore dopo aggiornamento core

### DISCUSSION THREAD

**Discussion by Gdii** [indomus/forum#1051] [GENERALE - Discussioni generiche]
Salve,
premettendo di essere assoluto neofita, chiedo aiuto per un problemino che mi e' sorto a seguito dell'aggiornamento alla versione "core-2021.11.4".
ho installato HA su un raspberry PI 4 come HassOS, da quando ho eseguito l'aggiornamento, nel registro mi compare:

Template variable error: 'value_json' is undefined when rendering '{{ value_json.last_update }}'
16:29:38 – (ERRORE) helpers/template.py - il messaggio si è verificato per la prima volta alle 08:19:31 e compare 1962 volte

...mentre i dettagli sono:

Logger: homeassistant.helpers.template
Source: helpers/template.py:1718
First occurred: 08:19:31 (1962 occurrences)
Last logged: 16:29:38

Template variable error: 'value_json' is undefined when rendering '{{ value_json.last_update }}'

Io non ho la piu' pallida idea di cosa si tratti...se qualcuno fosse cosi' gentile da spiegarmi un attimo, ne sarei molto grato.

grazie anticipate, Gdi.

**Comment by snakuzzo** [comment]
Leggi [QUI](https://www.home-assistant.io/docs/configuration/templating/#important-template-rules) - **PUNTO 2**

**Reply by Gdii** [reply]
Salve,
grazie mille per il suggerimento, credo di aver capito il senso, ma...non ho capito il punto in cui devo adare a cercare l'errore, da questa diagnosi, e' possibile risalire al punto da correggere??
Gdi.

**Reply by snakuzzo** [reply]
Cerca in tutta la tua configurazione dove hai scritto il template con `value_json.last_update`
Se non l'hai scritto tu ed è relativo a qualche entità creata dall'autodiscovery (es. zigbee2mqtt), non puoi farci niente e devi aspettare che sistemino

**Reply by Gdii** [reply]
Ahh, ok......
cosi' si spiega meglio....io ho guardato un po'ovunque, ma "value_json.last_update"...non l'ho trovato da nessuna parte....quindi, sono vittimma incolpevole ?! ha ha ha

**Reply by Gdii** [reply]
...se puo' essere un idizio, mi sembra impazzita l'integrazione " Samsung smart TV", che mi fa chiedere in continuazione l'autorizzazione all'associazione dallo schermo del Tv....sono andato nelle opzioni dell'integrazione, e ho disabilitato il polling...ho provato a reinstallarla...ma nulla....

**Reply by snakuzzo** [reply]
Non ho capito il nesso con quello che chiedevi all'inizio.
Quel warning è generato dall'integrazione Samsung Smart TV ?
Se così fosse, hai provato a tenere disabilitata la l'integrazione e vedere se continua a loggarti quel warning ?

**Reply by Gdii** [reply]
diciamo che me ne sono accorto guardando il registro, dopo aver fatto l'aggiornamento, e quindi credevo fosse appunto dipeso da quello...contestualmente, l'integrazione Samsung ha iniziato a fare i capricci, ma non avevo collegato le cose...adesso provo a disattivarla,
grazie.

**Reply by snakuzzo** [reply]
Ok...ma dubito abbia a che fare con quello...poi considera che stiamo parlando di un warning...e non di un errore

**Reply by Gdii** [reply]
...infatti, ho gia' provato....e non e' quello...
Beh, no, non e' un warning.....e'proprio un errore !!!

**Reply by Gdii** [reply]
mi viene un altro dubbio.....
io uso come gestore zigbee "principale" Zigbee home automation, con un'antenna conbee.....ma.....non riuscendo con quella a gestire il sensore di vibrazione aqara, ho avuto la brillante idea di installare anche zigbee2mqtt....con un'altra antenna, pero' tutto questo, ormai molto tempo fa....e fin'ora aveva sempre funzionato....potrebbe essere ???
